hasher/src/hasher/endpoints.py

```diff
@@ -57,3 +57,12 @@ async def hash_accession_number(message: str) -> Response:
 )
 async def hash_mrn(message: str) -> Response:
     return Response(content=generate_hash(message, 64), media_type="application/text")
+
+
+@router.get(
+    "/hash-image-identifier",
+    summary="Produce secure hash appropriate for an image identifier "
+    "(MRN + accession num concatenated)",
+)
+async def hash_image_identifier(message: str) -> Response:
+    return Response(content=generate_hash(message, 64), media_type="application/text")
```

Comment:
Oh this surprises me. Why are we making a seperate endpoint that has the same implementation as `hash`?

Reply:
Mainly I was following the existing pattern. But do the different types of input even need different hashing methods? I'm not sure what the requirements are here.

Reply:
Yeah I think the previous is a bit silly too. I'd remove and just use `/hash` I assume it was planned to do something like restricting the size of it like they did with accession number, but then that changed and they didn't remove the superflous extra endpoint

Reply:
Have removed the new endpoint. Shall we save the consolidation of the other endpoints for another PR?

Reply:
Maybe make a new issue so we don't forget / to discuss further?